workflow/cron/operator.go

```diff
@@ -7,20 +7,25 @@ import (
 	"sort"
 	"time"
 
+	"github.com/Knetic/govaluate"
```

Comment:
Is this a new dependency?

Reply:
No, its how When is evaluated in hooks as well

Reply:
No, it's an existing dep. But it is a deprecated dep as I wrote [above](https://github.com/argoproj/argo-workflows/pull/13474#discussion_r1732030191)